doc/api/buffer.md

```diff
@@ -286,13 +288,15 @@ function:
 
 * [`Buffer.from(array)`]
 * [`Buffer.from(buffer)`]
+<!--lint disable maximum-line-length-->
 * [`Buffer.from(arrayBuffer[, byteOffset [, length]])`][`Buffer.from(arrayBuffer)`]
```

Comment:
The sheer number of places that require these comments around them is indicative that perhaps 80 chars is just not feasible.

Reply:
I guess we could go ahead and deactivate linting for the whole `buffer.md` file for now?